test/lit/wasm-split/instrument-memory64.wast

```diff
@@ -33,7 +91,7 @@
 ;; CHECK-NEXT:   (block
 ;; CHECK-NEXT:    (i64.store align=1
 ;; CHECK-NEXT:     (local.get $addr)
-;; CHECK-NEXT:     (i64.const {{.*}})
+;; CHECK-NEXT:     (i64.const -6733849689813251748)
```

Comment:
I think this file has automatic changes we don't want?

Reply:
Yep, undoing 👍